modules/photo/include/opencv2/photo.hpp

```diff
@@ -202,7 +202,7 @@ CV_EXPORTS_W void fastNlMeansDenoisingColored( InputArray src, OutputArray dst,
 /** @brief Modification of fastNlMeansDenoising function for images sequence where consecutive images have been
 captured in small period of time. For example video. This version of the function is for grayscale
 images or for manual manipulation with colorspaces. For more details see
-<http://citeseerx.ist.psu.edu/viewdoc/summary?doi=10.1.1.131.6394>
+<https://ieeexplore.ieee.org/document/1577245>
```

Comment:
`ieeexplore.ieee.org` doesn't provide free access, so that link is almost useless. So we don't use this resource in OpenCV.

Is there something like `arxiv.org`?
If no, then this link may work: https://www.researchgate.net/publication/4215294_Denoising_image_sequences_does_not_require_motion_estimation

---

BTW, instead of link it is better to move citation to `opencv.bib` and `@cite Buades2005` here instead:

```
@inproceedings{Buades2005,
  author = {Buades, A. and Coll, Bartomeu and Morel, Jean-Michel},
  year = {2005},
  month = {10},
  pages = {70 - 74},
  title = {Denoising image sequences does not require motion estimation},
  volume = {2005},
  isbn = {0-7803-9385-6},
  journal = {IEEE International Conference on Advanced Video and Signal Based Surveillance - Proceedings of AVSS 2005},
  doi = {10.1109/AVSS.2005.1577245},
  url = {https://www.researchgate.net/publication/4215294_Denoising_image_sequences_does_not_require_motion_estimation}
}
```

Reply:
Thank you for the guidance. I could not find arxiv link, unfortunately. Requested changes are made using researchgate link 👍 @alalek